runelite-client/src/main/java/net/runelite/client/plugins/barrows/BarrowsBrotherSlainOverlay.java

```diff
@@ -89,6 +91,10 @@ public Dimension render(Graphics2D graphics)
 				.rightColor(rewardPercent >= 73.0f && rewardPercent <= 88.0f ? Color.GREEN : rewardPercent < 65.6f ? Color.WHITE : Color.YELLOW)
 				.build());
 
-		return panelComponent.render(graphics);
+		// Toggle for barrows overlay
+		if (config.showBarrowsOverlay())
```

Comment:
This isn't the right way to do a guard clause; we shouldn't even be running the code that updates `panelComponent` if `config.showBarrowsOverlay()` is false.